### PR TITLE
Add the 'all' flag when installing components through DISM in 2012 and a...

### PR DESCRIPTION
**Pull request by ksubrama**
...bove.

### CODE REVIEW

**Comment by ksubrama** [comment]
Fixes #102

**Comment by smurawski** [comment]
:+1:

File: libraries/helper.rb [hunk 1]
```diff
@@ -43,6 +43,22 @@ def self.older_than_windows2008r2?
         end
       end
 
+      def self.older_than_windows2012?
```
Comment (smurawski) [inline libraries/helper.rb]: This is an ugly helper and should have something in chef-client core for this.

Comment (jdmundrawala) [inline libraries/helper.rb]: I agree. Maybe you can use the os version and check for `<` that version. Perhaps something similar to https://github.com/chef/chef/blob/master/lib/chef/platform/query_helpers.rb#L31

**Pull request merged**